app/src/main/java/xyz/malkki/neostumbler/ui/screens/ReportsScreen.kt

```diff
@@ -82,15 +89,24 @@ fun ReportsScreen() {
     MLSWarningDialog()
 
     Column(modifier = Modifier.padding(16.dp)) {
+        ReportStats()
+        HorizontalDivider(
+            modifier = Modifier.padding(vertical = 16.dp),
+        )
+
+        Text(
+            text = stringResource(R.string.reports),
+            style = MaterialTheme.typography.titleMedium,
+        )
+        Box(modifier = Modifier.weight(1f)) {
+            Reports()
+        }
+
         Row(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
             ForegroundScanningButton()
-
             Spacer(modifier = Modifier.weight(1.0f))
-
             ReportUploadButton()
         }
```

Comment:
Let's keep the buttons at the top for now. I'm planning on doing a larger rework to this screen for the next major release, so I think it's better if the layout is not constantly changing